mopidy_alsamixer/mixer.py

```diff
@@ -24,40 +24,49 @@ def __init__(self, config):
         super().__init__()
         self.config = config
         self.cardindex = self.config["alsamixer"]["card"]
+        self.device = self.config["alsamixer"]["device"]
         self.control = self.config["alsamixer"]["control"]
         self.min_volume = self.config["alsamixer"]["min_volume"]
         self.max_volume = self.config["alsamixer"]["max_volume"]
         self.volume_scale = self.config["alsamixer"]["volume_scale"]
 
+        if self.cardindex is not None:
+            cardname = f"soundcard with index {self.cardindex:d}"
+        else:
+            cardname = f"soundcard with name '{self.device}'"
```

Comment:
I don't like variables that concatenate two words without an underscore (even though `cardindex` already does this). Also, the context here is clear enough, so I think `name` or `label` is descriptive enough. Also, the common text should be moved to where it is printed.

```suggestion
        label = (
            f"index {self.cardindex:d}"
            if self.cardindex is not None
            else f"name {self.device!r}"
        )
```